### DISCUSSION TITLE
Expo Router is moving upstream

### DISCUSSION THREAD

**Discussion by EvanBacon** [expo/router#820] [Announcements]
In order to start development on Expo Router v3, we've begun moving the contents of this repo to the upstream [expo/expo](https://github.com/expo/expo/) repo. The first major migration saw the docs move from the [GitHub pages](https://expo.github.io/router/docs) for this repo to the official [Expo docs](https://docs.expo.dev/routing/introduction/) in v2––the benefits here being proper vertical integration and more buy-in from the Expo team.

In https://github.com/expo/expo/pull/23725 we moved `expo-router`, `@expo/metro-runtime`, and `expo-head` to the expo/expo repo. The next step will be to create better e2e tests and community resources for filing issues. This migration will make it easier to test more scenarios, and drive better integration with Expo CLI and the Expo SDK (e.g. adding static font extraction to `expo-font`). We can also start onboarding more members of the Expo team to support and develop the project. 

This repo will start to wind down as we gear up for an exciting new chapter of file-based universal routing!

~ Evan Bacon

**Comment by axeldelafosse** [comment]
Nice work Evan!
